### ISSUE TITLE
Provide flexible class naming options (e.g. grass, broadleaf, clover)

### ISSUE THREAD

**Issue by mkutu** [Sydney-Informatics-Hub/Weed-AI#556]
I get the impression you want genus species for class name. Not sure if I'm missing this somewhere, but any thoughts on how a  class name should be given if multiple species were included in a single class? 

For example, we developed our dataset with three main plant classes (clover, broadleaf, and grass). We could provide genus species for clover, but the grasses and broadleaf would each have at least three species. Would you only want the dataset with clover (Trifolium repens)?

**Comment by jnothman** [comment]
Yeah, this thought is somewhat half-baked in implementation. Genus (or other taxonomic group) is supported instead of species, but we don't have a way to represent a class being a non-taxonomic group of weeds.
